traffic_ops/testing/api/v2/user_test.go

```diff
@@ -37,6 +39,9 @@ func TestUsers(t *testing.T) {
 		GetTestUsers(t)
 		GetTestUserCurrent(t)
 		UserTenancyTest(t)
+		if includeSystemTests {
+			UserRegistrationTest(t)
```

Comment:
since this force-deletes users, we should include a comment that this test needs to run last in the sequence, otherwise tests that come after it may fail, right?

Reply:
Yes. Included a comment about this in 53b2e5816.